pkg/commands/lsp/lsp.go

```diff
@@ -0,0 +1,122 @@
+/*
+Copyright 2023 VMware, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package lsp
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strconv"
+
+	"github.com/vmware-tanzu/apps-cli-plugin/pkg/apis/lsp"
+	"github.com/vmware-tanzu/apps-cli-plugin/pkg/cli-runtime"
+	"github.com/vmware-tanzu/apps-cli-plugin/pkg/source"
+)
+
+const errFormat = "%s\nErrors:\n- %s"
+
+type lspResponse struct {
+	Message    string `json:"message"`
+	StatusCode string `json:"statuscode"`
+}
+
+func GetStatus(ctx context.Context, c *cli.Config) (lsp.LSPStatus, error) {
+	r := &lspResponse{}
+	var localTransport *source.Wrapper
+	var resp *http.Response
+	var err error
+
+	if localTransport, err = source.LocalRegistryTransport(ctx, c.KubeRestConfig(), c.GetClientSet().CoreV1().RESTClient(), "health"); err != nil {
```

Comment:
If LSP is not installed what would be the response and error on CLI?

Reply:
check the [comment](https://github.com/vmware-tanzu/apps-cli-plugin/pull/524#issuecomment-1498240628) with examples output, there is the error when LSP not installed